base/common/src/main/java/com/netscape/cmsutil/http/HttpRequest.java

```diff
@@ -74,7 +74,10 @@ public void write(OutputStreamWriter writer)
             throw e;
         }
 
-        mLine = mMethod + " " + mURI + " " + Http.HttpVers;
+	String protocol = (getHeader("host") != null) ? Http.Vers1_1 : Http.HttpVers;
```

Comment:
We probably could use `Http.Vers1_0` instead of `Http.HttpVers` to make it more explicit, but either way is fine.